### PR TITLE
feat: Add slash commands

### PR DESCRIPTION
**Pull request by gsppe**
Adds support for / slash commands to execute operations directly without going through command detection (it is skipped).

Available commands (with shortcuts)

/text /t
/style /s
/ui /u
/delete /d

The branch is on top of #2472 because that PR changes a lot in the same file

### CODE REVIEW

**Comment by kof** [comment]
- [ ] We probably need to add info about it to the menu

File: apps/builder/app/builder/features/ai/ai-fetch-result.ts [hunk 1]
```diff
@@ -34,29 +34,56 @@ import untruncateJson from "untruncate-json";
 import { traverseTemplate } from "@webstudio-is/jsx-utils";
 import { RequestParamsSchema } from "~/routes/rest.ai._index";
 
+const slashCommands = {
+  text: copywriter.name,
+  t: copywriter.name,
```
Comment (kof) [inline apps/builder/app/builder/features/ai/ai-fetch-result.ts:39]: idk regarding shortcuts, do we need them? is it popular?

**Comment by istarkov** [comment]
I thought to add slash support through a textarea menu and pass command as standalone param, without text parsing in a fetch

**Comment by kof** [comment]
We need to think more about this feature, we have a few more ideas and implementation details, but this seems to be a mixture of the feature and other changes, I hope by closing this we are not missing out on some other important fixes @gsppe

**Pull request closed without merging**